package main;

option java_package = "com.gamebean.sanguocard.message.lua";
option java_outer_classname = "LuaProtos";

//----------LUA通用消息500
message LuaCommonRequest {
	//message id=500
	required int32 cmd = 1;
	optional bytes request = 2;
}

message LuaCommonResponse {
	//message id=500
	required int32 cmd = 1;
	optional bytes response = 2;
}

message LuaTestRequest {
	//message id=0
	optional int32 cmdId = 1;
	optional string info = 2;
}

message LuaTestResponse {
	//message id=0
	optional int32 cmdId = 1;
	optional string info = 2;
}